encrypted-media/scripts/playback-temporary-multikey-sequential.js

```diff
@@ -62,11 +63,12 @@ function runTest(config,qualifier) {
         }
 
         function onPlaying(event) {
-            assert_equals(_mediaKeySessions.length, 1, "Playback should start with a single key / session");
+            _playingCount++;
+            assert_equals(_mediaKeySessions.length, _playingCount, "Should get one 'playing' event per key / session played");
         }
 
         function onTimeupdate(event) {
-            assert_false(_waitingForKey, "Should not continue playing whilst waiting for a key");
+            assert_true(!_waitingForKey || _video.readyState == _video.HAVE_METADATA, "Should not continue playing whilst waiting for a key");
```

Comment:
Yes, we're required to fire 'timeupdate' when we transition from readyState HAVE_FUTURE_DATA or more to HAVE_CURRENT_DATA or less. I've added a comment.